src/main.js

```diff
@@ -4,7 +4,12 @@ var coverImage = document.querySelector(".cover-image");
 var coverTitle = document.querySelector(".cover-title");
 var coverDescriptor1 = document.querySelector(".tagline-1");
 var coverDescriptor2 = document.querySelector(".tagline-2");
+var homeButton = document.querySelector('.home-button');
 var showNewRandomCoverButton = document.querySelector('.random-cover-button');
+var saveCoverButton = document.querySelector('.save-cover-button');
+var viewSavedCoversButton = document.querySelector('.view-saved-button');
+var homeView = document.querySelector('.home-view');
+var savedView = document.querySelector('.saved-view');
 
 // We've provided a few variables below
 var savedCovers = [
```

Comment:
We need to see this section!

Reply:
We actually don't need to see the contents of the `savedCovers` array until iteration3! 😄

Reply:
Oh okay, I wasn't sure what was meant by "we should see the saved covers section"; sounds good then!